Gem/Code/multiplayersample_autogen_files.cmake

```diff
@@ -11,4 +11,16 @@ set(FILES
     ${LY_ROOT_FOLDER}/Gems/Multiplayer/Code/Include/Multiplayer/AutoGen/AutoComponent_Source.jinja
     ${LY_ROOT_FOLDER}/Gems/Multiplayer/Code/Include/Multiplayer/AutoGen/AutoComponentTypes_Header.jinja
     ${LY_ROOT_FOLDER}/Gems/Multiplayer/Code/Include/Multiplayer/AutoGen/AutoComponentTypes_Source.jinja
+    Source/AutoGen/NetworkAiComponent.AutoComponent.xml
```

Comment:
Can we glob anything ending with AutoComponent.xml? Provide a comment showing an example of how to subtract a file if anyone ever needs to.

Reply:
This function exactly as it did previously. I just moved AutoGen jinjas and xmls into a separate cmake.